src/main/resources/assets/tconstruct/book/fantastic_foundry/zh_cn/alloyer/fluid_cannon.json

```
@@ -0,0 +1,14 @@
+{
+  "title": "流体加农炮",
+  "item": "tconstruct:scorched_fluid_cannon",
+  "text": [
+    {
+      "text": "Thruul觉得他实在是太聪明了，能做出射出流体子弹攻击敌人的流体加农炮。但我的流体加农炮由钴制成，更加坚硬，射出的流体子弹更强；但唯一的缺陷是流体消耗速度有点快了。",
```

```suggestion
      "text": "Thruul拿他的流体加农炮朝敌人发射流体子弹来攻击，好像是什么了不得的事似的。但我的流体加农炮由钴制成，更加坚硬，射出的流体子弹更强；但唯一的缺陷是流体消耗速度有点快了。",
```
Polishing, not required.
~~Also not recommended.~~